docs/test_candlestick_agg.md

```diff
@@ -0,0 +1,90 @@
+# Candlestick Continuous Aggregation Tests
+
+## Setup table
+```SQL,non-transactional,ignore-output
+SET TIME ZONE 'UTC';
+CREATE TABLE stocks_real_time (
+  time TIMESTAMPTZ NOT NULL,
+  symbol TEXT NOT NULL,
+  price DOUBLE PRECISION NULL,
+  day_volume INT NULL
+);
+SELECT create_hypertable('stocks_real_time','time');
+CREATE INDEX ix_symbol_time ON stocks_real_time (symbol, time DESC);
+
+CREATE TABLE company (
+  symbol TEXT NOT NULL,
+  name TEXT NOT NULL
+);
+```
+
+## Setup Continuous Aggs
+```SQL,non-transactional,ignore-output
+CREATE MATERIALIZED VIEW cs
+WITH (timescaledb.continuous) AS
+SELECT time_bucket('1 minute'::interval, "time") AS ts,
+  symbol,
+  candlestick_agg("time", price, day_volume) AS candlestick
+FROM stocks_real_time
+GROUP BY ts, symbol;
+```
+
+## Insert data into table
+```SQL,non-transactional,ignore-output
+INSERT INTO stocks_real_time("time","symbol","price","day_volume")
+VALUES
+('2023-01-11 17:59:57-06','AAPL',133.445,NULL),
+('2023-01-11 17:59:55-06','PFE',47.38,NULL),
+('2023-01-11 17:59:54-06','AMZN',95.225,NULL),
+('2023-01-11 17:59:52-06','AAPL',29.82,NULL);
```

Comment:
I'd like to see at least three values per bucket, just so that we're seeing some more interesting cases as to which data is getting chosen.  It also might be worth making up some volume data so we can verify that's being updated correctly as well.  Finally, might be easier to verify the correctness of this if you use the same timezone in your input that you get in your output (`+00`).